src/components/icon/Icon.tsx

```diff
@@ -1,10 +1,11 @@
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
 import classnames from 'classnames';
+import { TkIcon } from  '@symphony-ui/uitoolkit-styles/dist/fonts/tk-icons';
```

Comment:
this import looks a bit weird to me :/ 
we should be able to do something like 
import {TkIcon} from '@symphony-ui/uitoolkit-styles'

Reply:
I agree with @symphony-adnane 
A question, maybe we should not provide them in the 'dist' folder, but at the root folder to be able to use the line suggested by @symphony-adnane 🤔